mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1208,9 +1208,9 @@ def Rock_AccelGemmOp:
     Rock_Op<"accel_gemm">,
     Arguments<(ins Index:$mRepeat,
                    Index:$nRepeat,
-                   MemRefRankOf<AccelArgTypes, [1]>:$matrixA,
-                   MemRefRankOf<AccelArgTypes, [1]>:$matrixB,
-                   MemRefRankOf<AccelResTypes , [1]>:$matrixC,
+                   Arg<MemRefRankOf<AccelArgTypes, [1]>, "source register A", [MemRead]>:$matrixA,
+                   Arg<MemRefRankOf<AccelArgTypes, [1]>, "source register B", [MemRead]>:$matrixB,
+                   Arg<MemRefRankOf<AccelResTypes , [1]>, "source register C", [MemWrite]>:$matrixC,
```

Comment:
I think that C gets updated by this op?

Reply:
You mean the operand is read and written? I don't think there is a `MemReadWrite` among the effects, but for my purposes, if something is read and the written, the `write` side effect is the one that matters.

Reply:
I claim that the side effects should be `[MemRead, MemWrite]` then.

`MemWrite` is a pure store operation.